src/SumoCoders/FrameworkCoreBundle/Resources/views/Form/fields.html.twig

```diff
@@ -69,29 +69,31 @@
 {% block date_widget %}
   {% spaceless %}
     {% if widget == 'single_text' %}
-      {% if datepicker is defined %}
+      {% if datepicker %}
```

Comment:
will datepicker always be defined? because `if datepicker` is not the same as `if datepicker is defined`. The former will throw an error if the datepicker variable isn't available

Reply:
datepicker is defined as a default in dateTypeExtension